aten/src/ATen/cuda/CUDAUtils.h

```diff
@@ -17,4 +17,18 @@ inline bool check_device(ArrayRef<Tensor> ts) {
   return true;
 }
 
+struct NoTF32Guard {
```

Comment:
Add a comment describing this struct with a brief example of how it's intended to be used.

Reply:
fixed